api/indexer.oas2.json

```diff
@@ -1195,6 +1245,16 @@
         },
         "is-frozen": {
           "type": "boolean"
+        },
+        "opted-in-at-round": {
+          "description": "Round during which the account opted into the asset.",
+           "type": "integer",
+           "x-algorand-format": "uint64"
+        },
+        "opted-out-at-round": {
+          "description": "Round during which the account opted out of the asset.",
+           "type": "integer",
+           "x-algorand-format": "uint64"
```

Comment:
I'm not exactly sure why we have AssetHolding and MiniAssetHolding. This one is matching AssetHolding above.